pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: cached_firestorage
 description: A Flutter utility that manages Firebase Storage download URLs and caches the results
-version: 0.1.8
+version: 0.1.9
```

Comment:
It's better to release a minor rather than a patch. Please edit this to be 0.2.0
After that LGTM